internal/app/i18n/bundle.en.go

```diff
@@ -0,0 +1,68 @@
+	GenServerError:                    "Unexpected error occurred while connecting to the generation server. Please try again later.\n",
+	GenServerTimeoutError:             "Timeout occurred while requesting the generation server. Please try again later.\n",
+	NetworkError:                      "Unexpected network error occurred while connecting to the generation server. Please check your Internet connection.\n",
+	InternalError:                     "An internal error occurred. Please file an issue on https://youtrack.jetbrains.com/newIssue?project=ktor\n",
+	ProjectDirExist:                   "The project directory %s already exists.\n",
+	NoPermsCreateProjectDir:           "Not enough permissions to create project directory %s.\n",
+	ProjectExtractError:               "Unable to extract downloaded archive to the directory %s.\n",
+	JdkExtractError:                   "Unable to extract downloaded JDK to the directory %s.\n",
+	DirAlreadyExist:                   "The directory already exists.",
+	UnableExtractJdk:                  "Unable to extract downloaded JDK.\n",
+	UnableDownloadJdk:                 "Unable to download JDK %s for %s %s\n",
+	JdkServerError:                    "Unexpected error occurred while connecting to a JDK server. Please try again later.\n",
+	JdkServerDownloadError:            "An error occurred while downloading from a JDK server. Please try again later.\n",
+	ChecksumVerificationFailed:        "Checksum verification for the downloaded JDK failed",
+	UnableMakeFileExec:                "Unable to make %s file executable.\n",
+	UnexpectedError:                   "Unexpected error occurred.\n",
+	UnexpectedErrorWithArg:            "Unexpected error %s\n",
+	UnableCreateStoreJdkDir:           "Unable to create a root directory %s to store downloaded JDKs.\n",
+	UnrecognizedFlagsError:            "Unrecognized flags: %s\n",
+	NoCommandError:                    "Expected a command",
+	CommandNotFoundError:              "Command '%s' not found\n",
+	CommandArgumentsError:             "Expected %d argument[s]: %s for the %s command\n",
+	ToRunProject:                      "To run the project use the following commands:\n\n",
+	JavaHomeJdkIdeaInstruction:        "You can also set the JAVA_HOME environment variable permanently or add the following JDK in the IntelliJ IDEA: \n",
+	ToolSummary:                       "Ktor is a tool mainly for generating Ktor projects.\n\n",
+	OptionsCaption:                    "The options are:",
+	CommandsCaption:                   "The commands are:",
+	VerifyingJdk:                      "Verifying %s...\n",
+	CreatingDir:                       "Creating directory %s\n",
+	Extracting:                        "Extracting %s to %s\n",
+	RequestGenServer:                  "Requesting generation server...",
+	ExtractingArchiveToDir:            "Extracting downloaded archive to directory %s\n",
+	ExtractProjectArchive:             "Extracting project archive... ",
```

Comment:
Is the empty space in the end on purpose? It also appears on lines 61,63 and 67

Reply:
Yes